File: app/src/main/java/com/recursia/popularmovies/utils/discover/PopularityDiscoverStrategy.java
package com.recursia.popularmovies.utils.discover;

import com.recursia.popularmovies.utils.NetworkUtils;

public class PopularityDiscoverStrategy implements DiscoverStrategy {

    @Override
    public String getSortBy() {
        return NetworkUtils.POPULARITY;
    }

    @Override
    public int getVoteCount() {
        return NetworkUtils.VOTE_COUNT_POPULARITY;
    }

}

File: app/src/main/java/com/recursia/popularmovies/utils/discover/PopularityDiscoverStrategy.kt
package com.recursia.popularmovies.utils.discover

import com.recursia.popularmovies.utils.NetworkUtils

class PopularityDiscoverStrategy : DiscoverStrategy {

    override val sortBy: String
        get() = NetworkUtils.POPULARITY

    override val voteCount: Int
        get() = NetworkUtils.VOTE_COUNT_POPULARITY

}
